src/ACL.cpp

```diff
@@ -238,3 +238,18 @@ QString ChanACL::permName(Perm p) {
 	}
 	return QString();
 }
+
+bool ChanACL::isPassword() const {
+	// A password is an ACL that applies to a group of the form '#<something>'
+	// AND grants 'Enter'
```

Comment:
This comment has to be corrected in order to correspond to the actual implementation. See #3928